clients/reality-data/package.json

```diff
@@ -55,7 +55,7 @@
     "@itwin/eslint-plugin": "workspace:*",
     "@itwin/core-geometry": "workspace:*",
     "@bentley/itwin-client": "workspace:*",
-    "@itwin/oidc-signin-tool": "workspace:*",
+    "@itwin/oidc-signin-tool": "3.0.0-dev.91",
```

Comment:
Let's wait to merge this until we can get a version released from the new location.